docs/Versioning.md

```diff
@@ -7,13 +7,13 @@ specification, MUST, SHOULD et al are used consistent with [RFC2119](https://too
 
 ONNX defines versioning policy and mechanism for three classes of entities:
 
-* The abstract graph model and the concrete format that serializes it. The graph model and format are versioned atomically and are referred to as the *IR version.* The IR version is  represented by the `ModelProto.ir_version` field.
-* Operators that are referenced by a given ONNX graph. The version of a given operator  is referred to as the *operator version*. The operator version is  represented by the `TBD` field.
+* The abstract graph model and the concrete format that serializes it. The graph model and format are versioned atomically and are referred to as the *IR version.* The IR version is represented by the `ModelProto.ir_version` field.
+* Operators that are referenced by a given ONNX graph.  The set of ONNX operators in scope and their semantics is versioned via the *ONNX operator version*. The ONNX operator version is represented by the `ModelProto.op_version` field.
```

Comment:
we should clarify whether ONNX and ONNX-ML (and any other profile) have independent versioning for IR and operators

Reply:
I think that it must version separately, it doesn't work reasonably otherwise.

Reply:
then we need a field in ModelProto to specify the profile